src/java/com/sarality/app/view/dialog/CustomViewDialogFragment.java

```diff
@@ -16,22 +16,30 @@
 /**
  * A DialogFragment that render a custom view inside a Dialog.
  *
+ * TODO(abhideep): Add Action support.
+ *
  * @author abhideep@ (Abhideep Singh)
  */
-public abstract class CustomViewDialogFragment<V extends View, T> extends BaseDialogFragment implements DataConsumer<T> {
+public abstract class CustomViewDialogFragment<V extends View, T> extends DialogFragment implements DataConsumer<T> {
```

Comment:
It wasn't really using any functionality from the BaseDialogFragment so it is best to just extend DialogFragments. The Action support for it will also look very similar to Fragment rather than DialogFragment